assets/app/view/game/corporation.rb

```diff
@@ -308,23 +310,22 @@ def render_revenue_history
 
       def render_abilities(abilities)
         attribute_lines = []
+        props = { style: { display: 'inline-block' } }
```

Comment:
i think we need a bit of margin / padding for the abilities

Reply:
I tried to add padding here but it just moved the ability lines a bit to the right, and kept Ability header as is, which looked fishy.

Any suggestion what should be done?